buildSrc/src/main/groovy/io/micronaut/gradle/MicronautWebsitePlugin.groovy

```diff
@@ -85,6 +87,16 @@ class MicronautWebsitePlugin implements Plugin<Project> {
             task.setDescription("Generates FAQ HTML - build/temp/faq.html ")
             task.setGroup(GROUP_MICRONAUT)
         })
+        project.tasks.register(TASK_GEN_MN_LOGOS, MicronautLogosTask, { task ->
+            Object extension = project.getExtensions().findByName(EXTENSION_NAME)
+            if (extension instanceof SiteExtension) {
+                SiteExtension siteExtension = ((SiteExtension) extension)
+                task.setProperty("logos", siteExtension.micronautLogos)
+                task.setProperty("output", siteExtension.output)
+            }
+            task.setDescription("Generates FAQ HTML - build/temp/mn-logos.html ")
```

Comment:
Wrong description. It does not generate a frequently ask question page

Reply:
Thanks, I totally missed that. Good catch.